app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -2,4 +2,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/home_screen_icon_background" />
     <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <monochrome android:drawable="@drawable/ic_monochrome" />
```

Comment:
```suggestion
    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
```

Can you check if that works as well?

Reply:
@Tgo1014 Did you check my suggestion?

Reply:
I've checked it now :D It also works, let me commit the change.